apps/daimo-web/package.json

```diff
@@ -31,7 +31,7 @@
   },
   "scripts": {
     "dev": "next dev -p 3001",
-    "build": "next build",
+    "build": "NODE_OPTIONS=\"--max-old-space-size=1536\" next build",
```

Comment:
👀

Reply:
this actually reduces the max heap
from the Render error before, looks like it defaulted to 2GB = machine memory size
> 254067 ms: Scavenge (reduce) 2000.3 (2074.4) -> 1999.8 (2074.4) MB, 21.43 / 0.00 ms  (average mu = 0.262, current mu = 0.291) allocation failure;